packages/x-date-pickers/src/AdapterMoment/index.ts

```diff
@@ -90,4 +94,12 @@ export class AdapterMoment
   public getWeekNumber = (date: defaultMoment.Moment) => {
     return date.week();
   };
+
+  public getWeekdays = () => {
```

Comment:
I think I'll write an RFC in the coming weeks, proposing to finally migrate all the adapters to our repository.
We are duplicating more and more logic...